packages/frontend/src/components/Form/ReportForm/ReportForm.tsx

```diff
@@ -87,15 +87,31 @@ const ReportForm: React.FC<ReportFormProps> = ({ closeModal, notifyParentAboutSu
 
     // Calculate possible stations based on entity, line, station, and search input
     const possibleStations = useMemo(() => {
-        let stations = allStations
+        // Sort station records from a station record [stationKey, station] array by station name
```

Comment:
I would remove this comment, imo if you feel like you need a comment that explains the function then the function should be renamed to be self explanitory. But in this case I think the function name was chosen very well and it clearly explains what the function does, so no need for the comment.

Reply:
Agreed!